### DISCUSSION TITLE
Solidity

### DISCUSSION THREAD

**Discussion by hussenk** [streetsidesoftware/vscode-spell-checker#2297] [Q&A]
Hello everyone,
Is this excitation works with Solidity?

because it doesn't work for me.

**Comment by Jason3S** [comment]
@hussenk,

To tell the spell checker to check Solidity files, please try this in VS Code:

1. Open a Solidity file.
2. Open the Command Palette
   <img width="283" alt="image" src="https://user-images.githubusercontent.com/3740137/196644617-dc96134c-7eba-42f2-8337-322412c4bbba.png">

4. Run the command: `Enable Spell Checking Document`
   <img width="371" alt="image" src="https://user-images.githubusercontent.com/3740137/196644110-9c6be4c8-0184-47a7-8fbf-103e62628536.png">
